src/index.js

```diff
@@ -166,11 +174,11 @@ export default class extends PureComponent {
 
     // Load the image
     this.image = new Image();
+    this.image.src = this.props.imgSrc;
```

Comment:
This was just recently changes to fix an issue (https://github.com/embiem/react-canvas-draw/pull/59).

Could you please remove this change of the `this.image.src = this.props.imgSrc;` line?